src/mongocxx/collection.cpp

```diff
@@ -316,6 +330,8 @@ stdx::optional<bsoncxx::document::value> collection::find_one_and_replace(
     options::return_document rd =
         options.return_document().value_or(options::return_document::k_before);
 
+    // TODO handle bypass document validation when C driver offers this for FAM
+    // @see https://jira.mongodb.org/browse/CDRIVER-945
```

Comment:
The '@thing' syntax is really just for our official header comments that get parsed into documentation.  For this it's fine to just do `// TODO handle bypass document validation when CDRIVER-945 is done` or something